config/enlightn.php

```diff
@@ -0,0 +1,208 @@
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Enlightn Analyzer Paths
+    |--------------------------------------------------------------------------
+    |
+    | The following array lists the "analyzer" paths that will be searched
+    | recursively to find analyzer classes. This option will only be used
+    | if the analyzers option above is set to the asterisk wildcard. The
+    | key is the base namespace to resolve the class name.
+    |
+    */
+    'analyzer_paths' => [
+        'Enlightn\\Enlightn\\Analyzers' => base_path('vendor/enlightn/enlightn/src/Analyzers'),
+        'Enlightn\\EnlightnPro\\Analyzers' => base_path('vendor/enlightn/enlightnpro/src/Analyzers'),
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Enlightn Base Path
+    |--------------------------------------------------------------------------
+    |
+    | The following array lists the directories that will be scanned for
+    | application specific code. By default, we are scanning your app
+    | folder, migrations folder and the seeders folder.
+    |
+    */
+    'base_path' => [
+        app_path(),
+        database_path('migrations'),
+        database_path('seeders'),
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Environment Specific Analyzers
+    |--------------------------------------------------------------------------
+    |
+    | There are some analyzers that are meant to be run for specific environments.
+    | The options below specify whether we should skip environment specific
+    | analyzers if the environment does not match.
+    |
+    */
+    'skip_env_specific' => env('ENLIGHTN_SKIP_ENVIRONMENT_SPECIFIC', false),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Guest URL
+    |--------------------------------------------------------------------------
+    |
+    | Specify any guest url or path (preferably your app's login url) here. This
+    | would be used by Enlightn to inspect your application HTTP headers.
+    | Example: '/login'.
+    |
+    */
+    'guest_url' => '/privacy',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Exclusions From Reporting
+    |--------------------------------------------------------------------------
+    |
+    | Specify the analyzer classes that you wish to exclude from reporting. This
+    | means that if any of these analyzers fail, they will not be counted
+    | towards the exit status of the Enlightn command. This is useful
+    | if you wish to run the command in your CI/CD pipeline.
+    | Example: [\Enlightn\Enlightn\Analyzers\Security\XSSAnalyzer::class].
+    |
+    */
+    // @phan-suppress-next-line PhanPluginDuplicateConditionalUnnecessary
+    'dont_report' => true === env('SKIP_HTTP_CHECKS', false) ? [
```

Comment:
Can this just be made unconditional?

Reply:
I will be coming back to tweak this later. I need to do a full round of deployments so the nginx config updates are applied, then I can enable that check by default.